bin/cml-send-comment.js

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env node
 
+const print = console.log;
```

Comment:
seems odd to alias if it's only used once in the whole file

Reply:
Unavoidable due to monkey-patching of `console.log` as per https://github.com/iterative/cml/pull/581#discussion_r653473819; let's move the discussion there.

Reply:
:upside_down_face: lol JS